vck-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/openid/RequestOptionUnitTest.kt

```diff
@@ -0,0 +1,38 @@
+package at.asitplus.wallet.lib.openid
+
+import at.asitplus.wallet.lib.data.ConstantIndex
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+import kotlin.random.Random
+
+class RequestOptionUnitTest : FreeSpec({
+    val scheme = ConstantIndex.AtomicAttribute2023
+    val representations = listOf(
+        ConstantIndex.CredentialRepresentation.PLAIN_JWT,
+        ConstantIndex.CredentialRepresentation.SD_JWT,
+        ConstantIndex.CredentialRepresentation.ISO_MDOC
+    )
+
+
+    representations.forEach { representation ->
+        "${representation.name}: RequestOptions -> InputDescriptor -> RequestOptions quasi equality" {
+            repeat(3) {
```

Comment:
Why repeat it three times?

Reply:
To get more permutations of requested attributes i.e. empty, all etc